config/initializers/apipie.rb

```diff
@@ -17,7 +17,8 @@
 
   substitutions = {
     'operatingsystem_families' => Operatingsystem.families.join(", "),
-    'providers' => ComputeResource.providers.join(', ')
+    'providers' => ComputeResource.providers.join(', '),
+    'default_nic_type' => InterfaceTypeMapper::DEFAULT_TYPE.humanized_name.downcase
```

Comment:
Change the key to a symbol, `:default_nic_type` and the substitution will work.  I've fixed the others in #2281.